content-src/components/Search/Search.js

```diff
@@ -259,8 +277,10 @@ const Search = React.createClass({
           {formHistory.map(suggestion => {
             const active = (this.state.activeSuggestionIndex === suggestionsIdIndex);
             const activeEngine = this.getActiveEngine();
-            return (<li key={suggestion} role="presentation">
+            const suggestionIndex = suggestionsIdIndex++;
+            return (<li key={suggestion} role="option">
                   <a id={"history-search-suggestions-" + suggestionsIdIndex++ }
```

Comment:
Gah, I was looking at the wrong thing. I see they go `*-1`, `*-3`, and so on. So it should probably be changed to look like line 298 below?